bindings/matlab/Visualization/plotMeshInWorld.m

```diff
@@ -0,0 +1,60 @@
+function [meshHandles,transform]=plotMeshInWorld(linkMeshInfo,w_H_link)
+% Gets the mesh information for each link in the model.
+%     - Iputs:
+%         - `linkMeshInfo` : An individual `linkMeshInfo` from the array output variable from `getMeshes` function.
+%         - `w_H_link` : Homogeneous transform from the world to the link
+%     - Outputs:
+%         - `meshHandles`  : Struct that contains all the created handles for the meshes. There can be more than one mesh for each link.
+%         - `transform`            : The transform object for the link
+% 
+% Author : Francisco Andrade (franciscojavier.andradechavez@iit.it)
+%
+% Copyright (C) 2019 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+% This software may be modified and distributed under the terms of the
+% GNU Lesser General Public License v2.1 or any later version.
+
+        
+% Default values
+meshColor= [0.1843    0.3098    0.3098];
+transparency=0.5;
+scaleRatio=1/1000;
```

Comment:
The scale is not always 1/1000, it is saved in the `ExternalMesh::scale` attribute and should be used the valueu for which to scale.

Reply:
I was concerned about that cause I assume is a parameter that can be changed from the softy exporting the meshes. Although I think most CAD programs have mm as default unit.

Reply:
I see that it returns a vector size 3. Is there a case where the scale can be different for each axis? Is it why we need 3 values for the scale?

Reply:
Done